src/tsickle.ts

```diff
@@ -19,7 +19,7 @@ import {ModulesManifest} from './modules_manifest';
 import {getEntityNameText, getIdentifierText, Rewriter, unescapeName} from './rewriter';
 import {containsInlineSourceMap, extractInlineSourceMap, parseSourceMap, removeInlineSourceMap, setInlineSourceMap, SourceMapper, SourcePosition} from './source_map_utils';
 import {createTransformerFromSourceMap} from './transformer_sourcemap';
-import {createCustomTransformers, noOpTransformer} from './transformer_util';
+import {createCustomTransformers, synthesizeCommentsTransformer} from './transformer_util';
```

Comment:
I'm not seeing a diff corresponding to this import's name being changed - unused import?

Reply:
Good catch, done.